routing/http/contentrouter/contentrouter.go

```diff
@@ -200,21 +200,32 @@ func (c *contentRouter) FindPeer(ctx context.Context, pid peer.ID) (peer.AddrInf
 	for iter.Next() {
 		res := iter.Val()
 		if res.Err != nil {
-			logger.Warnw("error iterating provider responses: %s", res.Err)
+			logger.Warnw("error iterating peer responses: %s", res.Err)
```

Comment:
Change to this:
```suggestion
			logger.Warnf("error iterating peer responses: %s", res.Err)
```
or to this:
```suggestion
			logger.Warnw("error iterating peer responses", "err", res.Err)
```

Reply:
Thanks, IIRC these were mostly copy-pasted from the provider record versions so there's probably a fix to be done there as well.